src/aks-preview/HISTORY.rst

```diff
@@ -14,6 +14,10 @@ Pending
 * Fix `--localdns-config` parameter to handle null values in JSON configuration files gracefully, preventing crashes when DNS override sections are null.
 * Add jwtauthenticator commands `az aks jwtauthenticator add/update/show/list/delete` to manage JWT authenticators for a managed cluster.
 
+18.0.0b41
++++++++
+* Fix an issue in disabling ACNS performance (--acns-datapath-acceleration-mode None).
```

Comment:
Please move the history notes currently listed under the "Pending" section (a few lines above) into your newly added version.